app/src/Liquidator.ts

```diff
@@ -24,7 +27,22 @@ export type HealthCheckResponse = {
   message: string;
 };
 
+enum LiquidationError {
+  Healthy = "healthy",
+  Grace = "grace",
+  Unknown = "unknown",
+}
```

Comment:
Why not use "Aloe: healthy" and "Aloe: grace" as the strings here, instead of having separate constants?

Reply:
good idea